README.md

```diff
@@ -9,33 +10,40 @@
 ![GitHub commits since latest release (by date)](https://img.shields.io/github/commits-since/Welcome-Bot/welcome-bot/latest)
 
 ## Commands
+
 See [docs/commands.md](docs/commands.md)
 
 ## Inviting the bot
+
 The bot requires the following permissions:
+
 - Read messages
 - Send messages
 - Read Message History
 
 To invite the bot with all the necessary required permissions mentioned above you can use this invitation link:
-> https://discord.com/oauth2/authorize?scope=bot&client_id=848459799783669790&permissions=68608
+> <https://discord.com/oauth2/authorize?scope=bot&client_id=848459799783669790&permissions=68608>
```

Comment:
```suggestion
> https://discord.com/oauth2/authorize?scope=bot&client_id=848459799783669790&permissions=68608
```

Reply:
ESlint shows this as violation. And this was the fix.

Reply:
Bro, say eslint to ignore readme files

Reply:
@abhijoshi2k